tests/components/yale_smart_alarm/conftest.py

```diff
@@ -0,0 +1,62 @@
+"""Fixtures for the Yale Smart Living integration."""
+from __future__ import annotations
+
+import json
+from typing import Any
+from unittest.mock import patch
+
+import pytest
+from yalesmartalarmclient.const import YALE_STATE_ARM_FULL
+
+from homeassistant.components.yale_smart_alarm.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry, load_fixture
+
+ENTRY_CONFIG = {
+    "username": "test-username",
+    "password": "new-test-password",
+    "name": "Yale Smart Alarm",
+    "area_id": "1",
+}
+OPTIONS_CONFIG = {"lock_code_digits": 6}
+
+
+@pytest.fixture
+async def load_int(hass: HomeAssistant, load_json: dict[str, Any]) -> MockConfigEntry:
```

Comment:
Please name this something other than `int` as I first read it as loading an integer